docs/user/boot_options/index.md

```diff
@@ -32,6 +32,22 @@ to extend.
   agama.config_url=usb:///agama.yaml
   ```
 
+- `agama.info`
+  Points agama to info file that contains additional agama settings. It is useful when more options
+  are provided to agama and typing during boot is unreliable.
+
+  ```text
+  agama.info=ftp://myserver/devel.info
+  ```
+
+  :::warning
+  Available from Agama12.
```

Comment:
I would omit this, because we are not putting this information for other options.

Reply:
yeap, but on other hand as first beta for SLES is out, we should somehow start indicating when it is available.